src/kbs1.jl

```diff
@@ -0,0 +1,91 @@
+"""
+    test1!(u::Word, v::Word, rs::RewritingSystem, o::Ordering)
+Adds a rule to a rewriting system (if necessary) that insures that there is
+a word derivable form two given words using the rules in rewriting system.
+See [Sims, p. 69].
+"""
+function test1!(u::Word, v::Word, rs::RewritingSystem, o::Ordering)
+    a = rewrite_from_left(u, rs)
+    b = rewrite_from_left(v, rs)
+    if a != b
+        lt(o, a, b) ? push!(rs, b=>a) : push!(rs, a=>b)
+    end
+end
+
+"""
+    overlap1!(i::Integer, j::Integer, rs::RewritingSystem, o::Ordering)
```

Comment:
same comment about the order of arguments.
as a name I suggest `isconfluent_local` (there will be `isconfluent(rws::RewritingSystem)`)